pkgs/applications/version-management/mercurial/default.nix

```diff
@@ -1,6 +1,6 @@
-{ stdenv, fetchurl, python, makeWrapper, docutils, unzip, hg-git, dulwich
-, guiSupport ? false, tk ? null, curses, cacert
-
+{ stdenv, fetchurl, python, makeWrapper, docutils, unzip, curses, cacert
+, guiSupport ? false, tk ? null
+, gitSupport ? false, hg-git ? null, dulwich ? null
 , ApplicationServices }:
 
```

Comment:
I'll add that as a separate commit, and also add in an assert for the `guiSupport` and `tk` as well. Will be in the same PR.